packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -156,19 +155,23 @@ exports.questions = {
 		validators: [new RequiredValidator()]
 	}),
 	representationUpload: new MultiFileUploadQuestion({
-		title: 'Upload representations from other parties',
-		question: 'Upload the representations',
-		fieldName: 'representation-upload',
-		validators: [new RequiredValidator('You must add your documents')]
+		title: '**************placeholder*********************',
+		question: '**************placeholder*********************',
+		fieldName: 'representation-upload'
```

Comment:
This content was just updated in another Pr for the representation upload ticket, think this is also a merge/rebase artifact: 
https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3054